attribution-graphs-frontend/attribution_graph/gridsnap/init-gridsnap.js

```
@@ -0,0 +1,190 @@
+    }
+  }
+
+  function renderPositions(active){
+    gridItemSel.call(renderGridItem, 'next')
+
+    if (active){
+      gridItemSel.filter(d => d == active).call(renderGridItem, 'cur')
+      previewSel.datum(active).call(renderGridItem, 'next')
+    } else{
+      if (!isFillContainer){
+        var maxY = Math.max(maxY, d3.max(gridData, d => d.next.y + d.next.h))
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)